### DISCUSSION TITLE
Welcome to the Sitefox forum

### DISCUSSION THREAD

**Discussion by chr15m** [chr15m/sitefox#6] [Announcements]
## 👋 Hello!

Here you can get help with Sitefox.

Please feel free to post about:

  * Any questions you have about using Sitefox.
  * Examples of things you have built with Sitefox.
  * If you are stuck on something.
  * If you have a feature idea.
  * If you just want to say hello.

In this space please be kind and assume best intentions from others.

**Comment by maxweber** [comment]
Hi, your project looks great 👍 I have two questions:

- Which license will the project have?

- Where do you usually deploy your sitefox sites?

**Comment by chr15m** [comment]
Hi @maxweber, thanks for checking out the project.

1. Whoops, I should have added a license file, sorry about that. I've just added the MIT license boilerplate to the project. Thanks for prompting me on that.

2. I host my Sitefox sites on a basic Ubuntu VPS using [Piku](https://piku.github.io/).
